examples/getting-started/basic-setup/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "getting-started-basic-setup",
+  "version": "1.0.0",
+  "license": "MIT",
+  "private": true,
+  "author": "",
+  "scripts": {
+    "start": "serve ."
```

Comment:
It would be cool if this is running in background and the user will end up with an empty terminal (maybe via new query argument `?terminal=new` or something like that)

Currently they will end up with `serve .` running and are unable to follow the next steps in the guide, which is running webpack. (It's not obvious that one can cancel the process, and that would also break the browser)

Reply:
I am not sure I follow. Are you suggesting to remove `serve` and not run anything for the first step? IMO, each step should have some visual outcome, no? That's why I added `serve` so the page is actually served in the browser. The first chapter doesn't tell the user what to do with it. Might be that I missed something.

Reply:
Running `serve .` is good. I was only asking to open another terminal, so the user can continue with the instructions in the guide.
They will be asked to run `npm install lodash` next and this would be harder if the terminal is already blocked by another process.

Reply:
My idea was that the guide can click the button and follow the guides instructions for that point on directly in the browser.

Reply:
Oh I see. I was thinking to basically create one example per chapter in the getting-started and see the result of the chapter. Because it has all the files they were asked to create. Or do you just mean the steps for the terminal then?

Reply:
> I was thinking to basically create one example per chapter in the getting-started and see the result of the chapter.

Yep

Reply:
Updated.